plugins/out_s3/s3.c

```diff
@@ -994,17 +999,35 @@ static int upload_data(struct flb_s3 *ctx, struct s3_file *chunk,
         }
     }
 
+    size_t payload_size = 0;
+    ctx->is_Compressed = FLB_FALSE;
```

Comment:
to match fluent bit style and the style of the other S3 code this should be `is_compressed`